frontend/public/components/marketplace/_marketplace.scss

```diff
@@ -0,0 +1,19 @@
+.co-marketplace-modal {
+  &__body {
+    display: flex;
```

Comment:
In order to accommodate the smallest mobile viewports, this two column layout is likely going to need to change (not to mention scrolling is not enabled).  [What I did for catalog overlay](https://github.com/openshift/console/blob/11475527c84ae08b88bbd2300494c84422ee6421/frontend/public/components/_catalog.scss#L107-L144) may be of interest.  Note:  we should have one shared solution when all is said and done.

Reply:
Makes sense! I actually haven't had a chance to discuss the mobile view with anyone on the UX team. @tlwu2013 are there any mobile mocks for the modal overlay?

@rhamilto I am happy to point to your scss for now, or we can hop on a call next week to refactor the code together in a future PR - what do you think?

Reply:
It's probably fine for yours to merge, and then I'll refactor the CSS in mine.